test/parallel/test-debugger-stuck-regression.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+
+const common = require('../common');
+
+const fixture = path.join(
+  common.fixturesDir,
+  'debugger-stuck-regression-fixture.js'
+);
+
+const args = [
+  'debug',
+  `--port=${common.PORT}`,
+  fixture
+];
+
+const TEST_TIMEOUT_MS = 4000;
```

Comment:
Nit: Maybe use `common.platformTimeout(4000)` so Raspberry Pi devices get a little extra time in CI.